### ISSUE TITLE
ZWJ missing from name ?

### ISSUE THREAD

**Issue by subins2000** [opendatakerala/map.opendatakerala.org#4]
Awesome work! Just checked it out. Found that the placenames in website has ZWJ missing in it. This gives incorrect place names. I checked OSM node, the chil used there is ല + ് + ZWJ : https://www.openstreetmap.org/relation/11312298

Maybe all chil in OSM names should be migrated to atomic chil ? Doesn't the current use of ZWJ in names of OSM affect searching ?

![Screenshot_20211031_214936](https://user-images.githubusercontent.com/2305675/139592800-8d71e790-669d-45f2-8dad-e41aef03d527.png)
![Screenshot_20211031_215001](https://user-images.githubusercontent.com/2305675/139592814-a3c44d03-c9ea-4a85-9cde-48f58fa1dff0.png)

**Comment by manojkmohan** [comment]
Yes. Known issue with Wikidata. Need to check all attributes with running a bot. Data entry issue. Its better with osm i think.
@naveenpf @jinoytommanjaly

**Comment by manojkmohan** [comment]
Ohh. OSM also have same issue!. Mostly the things came via wikidata. There is no normalization scrip working in wikidata like ml.wikipedia

**Comment by jinoytommanjaly** [comment]
Most of the labels of the panchayat were fetched into Wikidata from Malayalam wiki long before by bots or by users. So most of them will have ZWJ characters on the labels. In WD we can have one label for every entity as well as several names can be added to Aliases. So we can use the names without ZWJ characters in the label filed and with ZWJ characters as aliases.
